src/css/tabs/motors.css

```
@@ -39,6 +39,31 @@
     flex-grow: 1;
 }
 
+.tab-motors #dialogEscDshotDirection-closebtn {
+    margin-right: 0px;
+    margin-bottom: 0px;
+    position: absolute;
+    right: 0px;
+    bottom: 0px;
+}
+
+.tab-motors #dialogEscDshotDirection {
+    width: 400px;
+    height:440px
```

Found one spacing issue ;-) 
```suggestion
    height: 440px
```